src/views/shortcodes/my-attendance-list.php

```diff
@@ -22,6 +22,9 @@
 					<span class="datetime">(<?php echo $start_date; ?>)</span>
 				<?php endif; ?>
 			</a>
+			<a href="<?php echo esc_url( get_permalink( $id ). 'tickets' ); ?>" target="_blank">
+                		- <span><?php esc_html_e( 'View Tickets', 'event-tickets' ) ?></span>
```

Comment:
* :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed
* :no_entry_sign: Inline PHP statement must end with a semicolon

<sub>hash: 8e3bf4c6ad275ab7c2e009543425fef8</sub>

Reply:
```suggestion
                		- <span><?php esc_html_e( 'View Tickets', 'event-tickets' ); ?></span>
```